blueprints/validations.py

```diff
@@ -0,0 +1,49 @@
+"""Module for validation actions inside of Blueprints."""
+
+
+class NonPositiveValueError(Exception):
+    """Raised when a value is less than or equal to zero."""
+
+
+class NegativeValueError(Exception):
+    """Raised when a value is negative."""
```

Comment:
Does it add value to add these exceptions to an exception group?

Reply:
What do you mean?

Reply:
https://realpython.com/python311-exception-groups/